source/components/ui/msm-modal/msm-modal.directive.js

```diff
@@ -0,0 +1,46 @@
+(function() {
+  'use strict';
+
+  /**
+   * @ngdoc directive
+   * @name components.ui.msmModal
+   * @restrict 'A'
+   *
+   * @description Renders a styled modal
+   */
+  angular
+      .module('msm.components.ui')
+      .service('msmModalOkCancel', MsmModalOkCancel)
+      .service('msmModalSelectFromListing', MsmModalSelectFromListing);
+
+  function MsmModalOkCancel($modal) {
+    return {
+      open: function(controller, parameters, size) {
+        return $modal.open({
+          animation: true,
+          templateUrl: 'components/ui/msm-modal/modal-ok-cancel.html',
+          controller: controller,
+          size: size || '',
+          resolve: parameters,
+          windowClass: 'app-modal-window'
+        })
+      }
+    }
+  }
+
+  function MsmModalSelectFromListing($modal) {
```

Comment:
You *could* go with a modalFactory to DRY your code. It would get the templateUrl as the parameter and return the modal function.